charts/gardener-dashboard/values.yaml

```diff
@@ -22,7 +22,10 @@ resources:
 hosts:
   - dashboard.ingress.example.org
   - dashboard.example.org
-
+annotations:
```

Comment:
please put `annotations` under an `ingress` property, in case there is a need to customize the annotations of a different resource

Reply:
The `annotations` have been moved to the `ingress` property